mkosi/distributions/debian.py

```diff
@@ -221,3 +220,26 @@ def invoke_apt(
     )
 
     return bwrap(["apt-get", operation, *extra], apivfs=state.root, env=env | state.environment)
+
+
+def install_apt_sources(state: MkosiState, repos: Sequence[str]) -> None:
+    if not state.root.joinpath("usr/bin/apt").exists():
+        return
+
+    sources = state.root / "etc/apt/sources.list"
+    if not sources.exists():
+        with sources.open("w") as f:
+            for repo in repos:
+                f.write(f"{repo}\n")
+
+    # Already contains a merged tree of repo_dirs after setup_apt
+    workspace_sources_dir = state.workspace / "apt/sources.list.d"
```

Comment:
Let's create this directory unconditionally in the workspace so we don't have to check if it exists here. Then we can get rid of the workspace_sources_dir and sources_dir variables since both will only have one use.